/oak:
  ros__parameters:
    pipeline_gen:
      i_nn_type: spatial
      i_enable_rgbd: true
    stereo:
      i_subpixel: true
      i_publish_topic: false
    nn:
      i_enable_passthrough: true
      i_enable_passthrough_depth: true

